apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.ts

```diff
@@ -12,24 +11,48 @@ import * as Sentry from '@prairielearn/sentry';
 import * as opsbot from '../../lib/opsbot';
 import * as github from '../../lib/github';
 import { config } from '../../lib/config';
-import { CourseRequestSchema, IdSchema } from '../../lib/db-types';
+import { CourseRequestSchema, IdSchema, UserSchema } from '../../lib/db-types';
+import { RequestCourse } from './instructorRequestCourse.html';
 
 const router = express.Router();
 const sql = loadSqlEquiv(__filename);
 
 router.get(
   '/',
   asyncHandler(async (req, res) => {
-    res.locals.navPage = 'request_course';
-    res.locals.course_requests = await queryRows(
+    const course_requests = await queryRows(
       sql.get_requests,
       {
         user_id: res.locals.authn_user.user_id,
       },
       CourseRequestSchema,
     );
 
-    res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+    for (const cr of course_requests) {
+      cr['details'] = null;
+
+      if (cr.approved_by && ['approved', 'denied'].includes(cr.approved_status)) {
+        const approver = await queryRow(
+          'SELECT * FROM users WHERE user_id = $user_id',
+          {
+            user_id: cr.approved_by,
+          },
+          UserSchema,
+        );
```

Comment:
This is an FYI, since I believe the other option (new schema) is superior, but if you need something like this, the new objects in `models` have methods for this kind of search:
https://github.com/PrairieLearn/PrairieLearn/blob/563c91c35c001618e096b363b66a969aa889dff4/apps/prairielearn/src/models/user.ts#L7